airflow-core/src/airflow/cli/commands/config_command.py

```diff
@@ -306,13 +305,11 @@ def message(self) -> str | None:
         config=ConfigParameter("logging", "log_filename_template"),
         was_removed=True,
         remove_if_equals="{{ ti.dag_id }}/{{ ti.task_id }}/{{ ts }}/{{ try_number }}.log",
-        breaking=True,
```

Comment:
We probably want to leave this one - it isn't handled in configuration.py (yet?), but let's do these log_filename_template ones in a follow up.

Reply:
Added back the breaking flag to both of `logging.log_filename_template` `ConfigChange`  (two distinct `remove_if_equals` forms)